packages/server/database/migrations/20210301164434-renameFocusedPhaseItemIdToFocusedPromptId.ts

```diff
@@ -0,0 +1,46 @@
+import {R} from 'rethinkdb-ts'
+export const up = async function(r: R) {
+  try {
+    await r
+      .table('NewMeeting')
+      .filter({meetingType: 'retrospective'})
+      .update({
+        phases: r.row('phases').map((phase) => {
+          return r.branch(
+            phase('focusedPhaseItemId')
+              .default(null)
+              .ne(null),
+            phase
+              .without('focusedPhaseItemId')
```

Comment:
i had to test this to believe it works!
usually, you have to do the `merge` before the `without` otherwise `without` will remove the source value, but in this case, rethinkdb treats `phase` as immutable i guess. learned something new!